newIDE/app/src/UI/Search/UseSearchStructuredItem.js

```diff
@@ -86,6 +89,12 @@ export const filterSearchResults = <
 
   const startTime = performance.now();
   const filteredSearchResults = searchResults
+    .filter(
+      ({ item }) =>
+        !chosenItemCategory ||
+        item.category === chosenItemCategory ||
+        (chosenItemCategory === 'General' && !item.category)
+    )
```

Comment:
It's probably a bad idea to use `'General'` here as it's common to other searches like examples and resources.